docs/development/intro.rst

```diff
@@ -0,0 +1,66 @@
+.. include:: ../references.txt
+
+.. _development_intro:
+
+Gammapy development getting started
+===================================
+
+So you're considering making a contribution to Gammapy?
+
+This page is a 10 min tutorial introduction walking you through the steps involved,
+using one common development workflow.
+
+It doesn't explain in detail the basics, i.e. how git, Github, pytest, Sphinx, etc. work.
+This is covered by the Astropy developer documentation and many other online resources.
+
+It also doesn't explain how to work on several branches in parallel using e.g. conda or virtual environments.
+
+Get in touch
+------------
+
+The first step should be to get in touch via the Gammapy mailing list or a Github issue, and to
+let us know what you want to add. This will spare you the frustration of developing something
+that doesn't fit somehow and has to be re-worked, or is already in Gammapy in a different form
+or is not appropriate for inclusion at all.
+
+Get set up
+----------
+
```

Comment:
Maybe still add a link to http://docs.gammapy.org/en/latest/install.html at the beginning? To explain how to setup a an Anacaonda Python installation...